### ISSUE TITLE
How to use Jcommander CLI arguments in the Spark application

### ISSUE THREAD

**Issue by stym06** [GoogleCloudPlatform/spark-on-k8s-operator#1279]
My Spark application uses JCommander to parse CLI arguments, however here in the arguments field in the yml file, it's like an array which passes down to the spark-submit command like below

`exec /usr/bin/tini -s -- /opt/spark/bin/spark-submit --conf spark.driver.bindAddress=10.xx.xx.20 --deploy-mode client --properties-file /opt/spark/conf/spark.properties --class com.example.MyClass local:///opt/spark/custom/MyApp.jar '--table-type mysql' '--props local:///opt/spark/myapp/config/source.properties'`

basically it's attaching quotes to the arguments.
How do I use this?

**Comment by tiechengsu** [comment]
I ran into the same issue, how did you resolve this at the end?
